src/index.ts

```diff
@@ -0,0 +1,33 @@
+import Express from 'express'
+import { config } from 'dotenv'
+import * as HikingAPI from './services/Hiking'
+
+config()
+
+const app: Express.Application = Express()
+
+const PORT: string | number = process.env.PORT || 4000
+
+app.use(Express.json())
+
+app.get('/trails', async (req: Express.Request, res: Express.Response) => {
```

Comment:
please add a router files instead and add a function from a different file instead of an anonymous funciton